### PR TITLE
copyright-check: Added permissive ', LLC.' to parent company name.

### PR DESCRIPTION
**Pull request by therandomstring**
Signed-off-by: Bálint Horváth <bal.horv.98@gmail.com>

### CODE REVIEW

File: tests/copyright/check.sh [hunk 1]
```diff
@@ -377,7 +378,7 @@ is_balabit_copyright() {
  grep --quiet --extended-regexp "\
 ^\
 (Copyright \(c\) ([0-9, -]+) [^ <][^<]*<br>)*\
-Copyright \(c\) ([0-9, -]+) (Bala[bB]it)|(One Identity)<br>\
+Copyright \(c\) ([0-9, -]+) (Bala[bB]it)|((One Identity)(, LLC\.)?)<br>\
```
Comment (MrAnno) [inline tests/copyright/check.sh:381]: OI's legal recommendation is to use `One Identity LLC.` without the comma.
I think we should conform to that form.

An open-source example:
https://github.com/google/oss-fuzz/blob/master/projects/digest/fuzz_digest.py#L2

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by MrAnno** [comment]
(Kira is failing due to an unrelated issue.)

**Comment by MrAnno** [comment]
Thank you.

**Pull request merged**